src/platform/silabs/multi-ota/OTATlvProcessor.h

```diff
@@ -86,10 +112,16 @@ enum class OTAProcessorTag
 class OTATlvProcessor
 {
 public:
+    struct Descriptor
```

Comment:
Based on the python script, this was added for the TA image processing, should it be in the common processor or in the SiWx917 processor?